parts/windows/windowscsehelper.ps1

```diff
@@ -302,6 +305,11 @@ function Install-Containerd-Based-On-Kubernetes-Version {
     if (([version]$KubernetesVersion).CompareTo([version]$global:MinimalKubernetesVersionWithLatestContainerd) -ge 0) {
       $containerdPackage=$global:LatestContainerdPackage
       Write-Log "Kubernetes version $KubernetesVersion is greater than or equal to $global:MinimalKubernetesVersionWithLatestContainerd so the latest containerd version $containerdPackage is used"
+      $buildNumber = (Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").CurrentBuild

```

Comment:
1. Suggest to add a function to get BuildNumber and reuse it in Get-WindowsVersion.
2. Update ut